### ISSUE TITLE
feat: Create a single-line installer for the CLI

### ISSUE THREAD

**Issue by kerinin** [kaskada-ai/kaskada#251]
**Summary**
The install script we have for the CLI (https://kaskada.io/docs-site/kaskada/main/getting-started/hello-world-cli.html#installation) works really well, but it's a lot of code to copy-paste. Let's find a way to do something similar to https://brew.sh/ so users don't get distracted by the install details.

**Comment by bjchambers** [comment]
Specifically, I think this refers to the following install command they provide:

```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

It looks like what they did is have an install script in their repo, and then just download it (with curl) and then run it with bash.

So, it seems like the tasks here are:

- [ ] Create an `install_cli.sh` script somewhere in the repo.
- [ ] Create (and test) an install command similar to the above that uses it.

One potential difficulty is needing to make that work for Windows (powershell)? Do we feel that is necessary at this time, or is the single `bash` based version similar to brew sufficient?

**Comment by kerinin** [comment]
Let's start with bash - we can add a powershell version in the future.

**Comment by therapon** [comment]
Yes that approach requires we store the shell script in a repo and provide a URL to grab it. A first step could be to take the long command we have not and wrap it in a shell script :) 

But with `bash` (or any other shell/language we decide to use) we can be a little more user friendly when something goes wrong.

**Comment by therapon** [comment]
Another option is to use python instead of bash. We can have the same python code in the Kaskada Python client that performs the checks and installation shared with the one-line install instructions. 

This approach relies on the user having python installed, but, on the plus side we might get it working for all OSes.
